packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -55,9 +58,12 @@ export class TestAccount {
 		await loginPage.visit();
 		await loginPage.logInWithCredentials( this.credentials.username, this.credentials.password );
 
-		const verificationCode = this.getTOTP();
-		if ( verificationCode ) {
-			await loginPage.submitVerificationCode( verificationCode );
+		// Handle possible 2FA.
+		if ( this.credentials.totpKey ) {
+			await loginPage.submitVerificationCode( this.getTOTP() );
+		}
+		if ( this.credentials.smsNumber ) {
+			await loginPage.submitVerificationCode( await this.getSMSOTP() );
```

Comment:
This logic is predicated on the current implementation of 2FA where the active form of 2FA is mutually exclusive; that is, only either SMS or TOTP may be active at any given time.

Reply:
Should this be an `else if`? It's semantics and I sort of get why you wouldn't but submitting both codes doesn't make sense either.

Reply:
Addressed in 2f93fdf836bdf423be0769629391e14c14208962.